app/converters/gbfs_lime_remove_stations.py

```diff
@@ -0,0 +1,35 @@
+"""
+MobiData BW Proxy
+Copyright (c) 2023, binary butterfly GmbH
+All rights reserved.
+"""
+
+from typing import List, Union
+
+from app.base_converter import BaseConverter
+
+
+class GbfsLimeRemoveStationsConverter(BaseConverter):
+    hostnames = ['data.lime.bike']
+
+    def convert(self, data: Union[dict, list], path: str) -> Union[dict, list]:
+        if not isinstance(data, dict) and not path.endswith('/gbfs.json'):
+            return data
+
+        if not isinstance(data, dict) or 'data' not in data or not isinstance(data['data'], dict):
+            return data
+
+        for language in data['data']:
+            new_feeds = []
+            if 'feeds' not in data['data'][language] or not isinstance(data['data'][language]['feeds'], list):
+                continue
+
+            for feed in data['data'][language]['feeds']:
+                if not isinstance(feed, dict) or 'name' not in feed or feed['name'] in ['station_information', 'station_status']:
```

Comment:
Could be simplified to `if not isinstance(feed, dict) or feed.get('name') in ['station_information', 'station_status']:`